src/kakarot/instructions/system_operations.cairo

```diff
@@ -272,7 +274,7 @@ namespace SystemOperations {
         return evm;
     }
 
-    // @notice CALL operation.
+    // @notice CALL operation. Message call into an account.
     // @dev
```

Comment:
maybe add here `@dev we don't pop the two last arguments (ret_offset and ret_size) to get them at the end of the CALL. These two extra stack values need to be cleard if the CALL early return without reverting (value > balance, stack too deep).`